open_fdd/desktop_bridge/server.py

```
@@ -0,0 +1,514 @@
+        try:
+            return services.ingest.ingest_csv(csv_path=csv_path, site_id=body.site_id, source=body.source)
+        except FileNotFoundError as exc:
+            raise HTTPException(status_code=400, detail=str(exc)) from exc
+        except ValueError as exc:
+            raise HTTPException(status_code=400, detail=str(exc)) from exc
+
+    @app.post("/ingest/csv/upload")
+    async def ingest_csv_upload(
+        site_id: str = Form(...),
+        source: str = Form("csv"),
+        file: UploadFile = File(...),
```

Ruff (0.15.12)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)